understanding/20/pause-stop-hide.html

```diff
@@ -110,58 +88,58 @@ <h2>Intent of Pause, Stop, Hide</h2>
          moves the focus away) would not be considered a "mechanism for the user to pause"
          because it makes the page unusable in the process and would not meet this SC.
       </p>
-      
-      <p>It is important to note that the terms "blinking" and "flashing" can sometimes refer
-         to the same content.
-      </p>
-      
+
+      <div class="note">
+         <p>This Success Criterion is specifically concerned with moving, blinking, scrolling, and
```

Comment:
Should Success Criterion be lower case here?

Reply:
I suspect that will be caught in the other PR.